botocore/__init__.py

```diff
@@ -14,10 +14,20 @@
 
 import logging
 import os
+import pathlib
 import re
 
 __version__ = '1.27.20'
 
+try:
+    # only available in python 3.9+
+    from importlib.resources import files
+
+    BOTOCORE_ROOT = files('botocore')
+except ImportError:
+
+    BOTOCORE_ROOT = pathlib.Path(os.path.dirname(os.path.abspath(__file__)))
+
```

Comment:
Same note here, no try/except. Just if/else on `HAS_ZIP_SUPPORT`.

Reply:
not sure i understand this comment. this try/except establishes whether or not `HAS_ZIP_SUPPORT` is true or false. Did you want it defined somewhere else, perhaps `compat.py` and imported here?